Tests/Functional/FailingTestSuite/main.swift

```diff
@@ -10,9 +10,7 @@
 
 class PassingTestCase: XCTestCase {
     static var allTests: [(String, PassingTestCase -> () throws -> Void)] {
-        return [
-            ("test_passes", test_passes),
-        ]
+        return [("test_passes", test_passes)]
```

Comment:
I actually prefer the former syntax. Listing each element on a separate line and including a trailing comma for each element ensures the diff is small when an element is added or removed.

But my preference notwithstanding, once again the convention for listing test cases appears to be one per line. I think convention trumps preference in this case.